Main.cs

```diff
@@ -1,93 +1,211 @@
 using System;
 using System.Collections.Generic;
 
-// Author: Sonia Álvarez
-
```

Comment:
**Error:** ¿Por que elimináis aquí la autoría?

Reply:
Porque la autoría es de todo el grupo